ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -104,3 +104,38 @@ def test_torch_softmax(
         dim=axis,
         dtype=dtypes[0],
     )
+
+
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        )
+    ),
+    num_positional_args=helpers.num_positional_args(fn_name="gelu"),
+    native_array=st.booleans(),
+    approximate=st.sampled_from(["none", "tanh"])
+)
+def test_torch_gelu(
+    dtype_and_x,
+    num_positional_args,
+    native_array,
+    approximate,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=False,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_tree="nn.functional.gelu",
```

Comment:
here fn_tree should be only = "gelu" according to the pytorch example in this (https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html) document.

Reply:
it doesn't work using just gelu, because I already check the fn_tree and I print it for debugging it calls the pytorch that's why sigmoid or softmax have no error on fn_tree, because sigmoid and softmax using torch.sigmoid and torch.softmax, but when it implemented on gelu, it use torch.nn.functional.gelu

Reply:
Oh, thanks , that was my misunderstanding then.